### DISCUSSION TITLE
[SOLVED] Lesson 7:  Import Error: cannot import name 'Lottery' from 'brownie'

### DISCUSSION THREAD

**Discussion by engfuentes** [smartcontractkit/full-blockchain-solidity-course-py#247] [Q&A]
Hello!

I am trying to run the deploy_lottery script. When I run in the terminal _brownie run scripts/deploy_lottery.py_ I get the following error:

![image](https://user-images.githubusercontent.com/73963334/138514402-ced94bae-9038-47bf-8f3b-ec8d53b540f8.png)

This is the script

![image](https://user-images.githubusercontent.com/73963334/138514678-029adaf1-9eb1-4365-b656-42d4d60ba953.png)

There aren´t any error when I compile the contracts.

**Comment by cromewar** [comment]
Please delete the `build` folder, then compile again and give it a try.

**Reply by engfuentes** [reply]
Thanks for the help, that was not the problem. I found that the problem was in Lottery.sol contract:
![image](https://user-images.githubusercontent.com/73963334/138567825-fc1867ed-236d-487e-ab71-9dd3362c80ec.png)

I wrote "contract lottery".... and it must be "contract Lottery"

**Reply by cromewar** [reply]
I'm glad you solved your problem :D
